sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/InMemoryPartitionManager.java

```diff
@@ -20,30 +20,52 @@
     private final Map<String, PartitionOwnership> partitionOwnershipMap = new ConcurrentHashMap<>();
     private final ClientLogger logger = new ClientLogger(InMemoryPartitionManager.class);
 
+    /**
+     * {@inheritDoc}
+     *
+     * @param eventHubName The Event Hub name to get ownership information.
+     * @param consumerGroupName The consumer group name.
```

Comment:
I would say that this adds no value.   Why does the user of the library care about the consumer group?  What's its purpose in this context?   If those are self-evident, then this is likely not a helpful comment.